unstoppable-ios-app/domains-manager-ios/Modules/Messaging/ChatsList/Views/ChatsListNavigationView.swift

```diff
@@ -237,15 +237,17 @@ struct ChatsListNavigationView_Previews: PreviewProvider {
             let wallet = WalletDisplayInfo(name: "name.x",
```

Comment:
Is it mocked data? Such things should not be in prod code

Reply:
This is SwiftUI preview, they're available only from Xcode